src/PhpGenerator/Method.php

```diff
@@ -107,6 +118,7 @@ public function __toString()
 			. ($this->name ? ' ' . $this->name : '')
 			. '(' . implode(', ', $parameters) . ')'
 			. ($this->uses ? ' use (' . implode(', ', $uses) . ')' : '')
+			. ($returnType ? ' : ' .$returnType : '')
```

Comment:
I checked PHP and it uses `x(): y` (ie. `Declaration of Bar::a(): string must be compatible with Foo::a(): DateTime`)

I will fix it later today, together with the missing space after dot.